examples/next/docs/js/demo/src/index.js

```diff
@@ -12,10 +12,13 @@ import {
   changeCheckboxCell
 } from "./hooksCallbacks";
 
+const urlParams = new URLSearchParams(location.search.replace(/^\?/, ""));
+const isArabicExample = urlParams.get("arabicExample") === "1";
 const example = document.getElementById("example");
 
 new Handsontable(example, {
-  data,
+  data: isArabicExample ? generateArabicData() : data,
```

Comment:
In the `arabicExample`, the demo should also present the Arabic UI, as in https://github.com/handsontable/handsontable/blob/d8d13ff2ecf76279f9393c1617f59653b6de06a4/examples/next/docs/js/demo/src/index.js#L74

Reply:
I've added the `language` option within 1cc365d8a3b4badafdc2fa8159ee4a4193d306dc